doc/news/changes/minor/20240728AE7TB99

```diff
@@ -0,0 +1,4 @@
+Uniformly apply the practice of importing the 'dealii' namespace
+only within the StepXX namespace across all tutorial programs.
+<br>
+(Lóránt Hadnagy, 2024/07/28)
```

Comment:
```suggestion
(Hadnagy Lóránt, 2024/07/28)
```

;)

Reply:
Should I change it back then? It doesn't matter to me, but I want it to be consistent ^^

Reply:
I changed it back for consistency's sake.